src/Sylius/Bundle/AdminBundle/Resources/views/Address/Grid/Field/logData.html.twig

```diff
@@ -0,0 +1 @@
+{% for key, field in data if field is not null %}{{ key }}: {{ field }} {% endfor %}
```

Comment:
We could, however it's only a cell in a table which does not contains so much data. I just don't like how table looks, when only one cell is massive and others are have one-line text 😕  🐃